xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/hierarchy.vm

```diff
@@ -33,7 +33,8 @@
     ##
     ## Default mode: display the hierarchy for the current page
     ##
-    #hierarchy($NULL {'id': 'hierarchy', 'limit': 5, 'treeNavigation': true})
+    #hierarchy($NULL {'id': 'hierarchy',
+    'limit': 5, 'treeNavigation': true})
```

Comment:
The additional line break seems to be the only change now, I think it should be removed now.